### PR TITLE
Improve security bulletin 001

### PR DESCRIPTION
**Pull request by israel-hdez**
* Fix mitigation script
* Add detection script
* Fix wording in a sentence
* Add paragraph about the secondary security issue

### CODE REVIEW

**Comment by israel-hdez** [comment]
@jshaughn Just to have a better bulletin.

File: content/news/security-bulletins/KIALI-SECURITY-001.adoc [hunk 1]
```diff
@@ -5,23 +5,61 @@ draft: false
...
+* **Affected Releases**: 0.4.0 to 1.15.0
 * **Impact Score**: link:https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator?vector=AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:H/A:H&version=3.1[9.4 - AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:H/A:H]
 
 A vulnerability was found in Kiali allowing an attacker to bypass the
 authentication mechanism. Currently, Kiali has four authentication mechanisms:
 _login, token, openshift_ and _ldap_. All are vulnerable.
 
-The vulnerability would let an attacker to build forged credentials and use
-them to gain unauthorized access to Kiali.
+The vulnerability lets an attacker to build forged credentials and use them to
```
Comment (jshaughn) [inline content/news/security-bulletins/KIALI-SECURITY-001.adoc]: to build -> build

File: content/news/security-bulletins/KIALI-SECURITY-001.adoc [hunk 2]
```diff
@@ -5,23 +5,61 @@ draft: false
...
+gain unauthorized access to Kiali.
+
+Additionaly, it was found that Kiali credentials aren't being validated
```
Comment (jshaughn) [inline content/news/security-bulletins/KIALI-SECURITY-001.adoc]: Additionaly -> Additionally
aren't -> were not

File: content/news/security-bulletins/KIALI-SECURITY-001.adoc [hunk 3]
```diff
@@ -5,23 +5,61 @@ draft: false
...
+properly. Depending on the authentication mechanism configured in Kiali, this
+may facilitate unauthorized access into Kiali with forged and/or invalid
```
Comment (jshaughn) [inline content/news/security-bulletins/KIALI-SECURITY-001.adoc]: may -> could

**Comment by israel-hdez** [comment]
@jshaughn Fixed!

**Pull request merged**